src/govuk/common.mjs

```diff
@@ -129,3 +129,51 @@ export function extractConfigByNamespace (configObject, namespace) {
   }
   return newObject
 }
+
+/**
+ * Normalise data
+ *
+ * 'If it looks like a duck, and it quacks like a duck…' 🦆
+ *
+ * If the passed value looks like a boolean or a number, convert it to a boolean
+ * or number.
+ *
+ * Designed to be used to convert config passed via data attributes (which are
+ * always strings) into something sensible.
+ *
+ * @param {String} value - The value to normalise
+ * @returns {(String|Boolean|Number)} Normalised data
+ */
+export function normaliseData (value) {
```

Comment:
Maybe we want to normalise unhandled values to an empty string?

(Guarding for undefined via `typeof value === 'string'`)

```suggestion
export function normaliseData (value) {
  if (typeof value !== 'string') {
    return value
  }

  // Allow TRUE, True, FALSE, False etc?
  value = value.trim().toLowerCase()
    
  if (['true', 'false'].indexOf(value) !== -1) {
    return value === 'true'
  }
  
  // Allow finite negatives without Math.abs()?
  if (isFinite(value)) {
    return Number(value)
  }
  
  return value
}
```